src/Pagination.jsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
+import { connect } from 'react-redux'
+import { changePage } from './actions/paging'
 
-export default class Pagination extends React.Component {
+export class Pagination extends React.Component {
   constructor(props) {
     super(props);
     this.state = this._calculatePages(props);
```

Comment:
I since page is calculated from `from` and `size` I thought it was better to store those and keep the primitives in the state tree.

Sort of like not keeping calculated columns in a DB